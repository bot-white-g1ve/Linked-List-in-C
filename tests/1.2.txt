NEW 1
a
VIEW 0
NEW 2

 6e-1
VIEW 1
NEW 3
   
   0.06  
a
VIEW 2
NEW 1
{
VIEW 3
VIEW ALL