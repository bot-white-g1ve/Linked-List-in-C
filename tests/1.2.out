List 0: a
a
List 1:  -> 0.60
 -> 0.60
List 2:     -> 0.06 -> a
    -> 0.06 -> a
INVALID COMMAND: NEW
INVALID COMMAND: VIEW
Number of lists: 3
List 0
List 1
List 2
